cpp/open3d/core/ParallelFor.h

```diff
@@ -79,10 +81,13 @@ void ParallelForCPU_(const Device& device, int64_t n, const func_t& func) {
         return;
     }
 
-#pragma omp parallel for num_threads(utility::EstimateMaxThreads())
-    for (int64_t i = 0; i < n; ++i) {
-        func(i);
-    }
+    tbb::parallel_for(tbb::blocked_range<int64_t>(0, n, 32),
+                      [&func](const tbb::blocked_range<int64_t>& range) {
```

Comment:
Also, avoid using "magic numbers" (32). I think you have a `GetDefaultChunkSize()` function.

Reply:
It will use up to the number of threads in task arena that called it. As for the chunk size, see [my other comment](https://github.com/isl-org/Open3D/pull/6626#discussion_r1505092754).